Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UnityPlayerBuildTools.cs

```diff
@@ -163,7 +163,7 @@ public static async void StartCommandLineBuild()
                 switch (EditorUserBuildSettings.activeBuildTarget)
                 {
                     case BuildTarget.WSAPlayer:
-                        success = await UwpPlayerBuildTools.BuildPlayer(new UwpBuildInfo(true) { BuildAppx = true });
+                        success = await UwpPlayerBuildTools.BuildPlayer(new UwpBuildInfo(true));
                         break;
                     default:
                         var buildInfo = new BuildInfo(true) as IBuildInfo;
```

Comment:
Why does this switch statement exist? It seems like this code is saying:

````
if (unity platform is UWP)
     build with default options
else
    parse command line options
    build with options configured by command line
````

Why wouldn't we want to use the command-line options for BuildTarget.WSAPlayer? Perhaps there's something I'm not understanding about this? Or could we remove the switch statement and replace it with always doing the code in the default case?

Reply:
I believe this question leads to the reason for having a switch:
> Why wouldn't we want to use the command-line options for BuildTarget.WSAPlayer?

So we do, but UwpPlayerBuildTools has to parse additional options specific to UWP builds. If you look at `UwpPlayerBuildTools.ParseBuildCommandLine`, you'll see this specific implementation.

The intention behind having a switch statement is that in future we may also have other build targets that have build steps or options that are specific to that target. When we provide build tools for those targets we can simply add an additional case to this switch statement. 

@StephenHodgson can correct me if I'm wrong.

Reply:
Aha! I hadn't noticed that UwpPlayerBuildTools.BuildPlayer calls its own ParseBuildCommandLine method while UnityPlayerBuildTools.BuildUnityPlayer doesn't call ParseBuildCommandLine so it has to be performed separately in the default case, but shouldn't be called in the WSAPlayer case. Thanks for explaining!